openslides_backend/action/actions/list_of_speakers/delete_all_speakers.py

```diff
@@ -30,3 +34,11 @@ def get_updated_instances(self, action_data: ActionData) -> ActionData:
                 yield from [
                     {"id": speaker_id} for speaker_id in list_of_speakers["speaker_ids"]
                 ]
+
+    def get_meeting_id(self, instance: Dict[str, Any]) -> int:
```

Comment:
is this necessary? THis should behave exactly like https://github.com/OpenSlides/openslides-backend/blob/master/openslides_backend/action/action.py#L167

Reply:
Yes, it is needed. It doesn't behave like the overwritten code.  In delete_all_speakers  he model is Speaker() and the id is from ListofSpeaker. That cannot work.

Reply:
Ah, now I see it, the `model = Speaker()`, right?

Reply:
Yes, but I would suggest to move this to a mixin/common super class to deduplicate the code